pkg/registries/docker/metadata_v1.go

```diff
@@ -137,38 +130,32 @@ func (r *Registry) handleV1ManifestLayer(remote string, ref digest.Digest) (*sto
 		// See github.com/moby/moby/image/image.go
```

Comment:
Should we update this comment to point to the opencontainers thing?

Reply:
I removed this comment as it's not helpful at all. What should I see there?